api/trace/api.go

```diff
@@ -100,7 +84,7 @@ type Span interface {
 
 	// End completes the span. No updates are allowed to span after it
 	// ends. The only exception is setting status of the span.
-	End(options ...EndOption)
+	End(options ...SpanOption)
```

Comment:
Should `End()` document which of the SpanOptions can take effect? It looks like only the timestamp.

Reply:
I think it should, but only on the implementation side. I added comments for [the default SDK](https://github.com/open-telemetry/opentelemetry-go/pull/1108/files#diff-06fc1a413b40f256294db86b8f7a33b2R111-R117) to specify exactly the implementation details that you mentioned. It seemed incorrect to place restrictions what is supported from the implementation level into the API.

Reply:
Going to resolve. Please let me know if this was in error.